filesystem/azure.go

```diff
@@ -0,0 +1,196 @@
+package filesystem
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"github.com/MSOpenTech/azure-sdk-for-go/storage"
```

Comment:
the 3rd party imports should be separated from the standard one.

i suggest you use goimports to manage it.
```
// standard
"fmt"

// in pkg
"taskgraph/abc"

// out pkg
"azure/abc"
```